### PR TITLE
[4.0] Update com_finder.sys.ini

### PR DESCRIPTION
**Pull request by tecpromotion**
### Summary of Changes

Deletion of multiple used strings and alphabetical sorting adjusted.
@alikon I think here 168748a6e8688cc997f67f4de3639839744b154b an error has crept in.

### Testing Instructions

.

### Expected result

.

### Actual result

.

### Documentation Changes Required

.

### CODE REVIEW

File: administrator/language/en-GB/com_finder.sys.ini [hunk 1]
```diff
@@ -5,33 +5,19 @@
...
-COM_FINDER_INDEX_VIEW_DEFAULT_TITLE="Smart Search Indexed Content"
 COM_FINDER_INDEX_VIEW_DEFAULT_DESC="Display indexed content."
+COM_FINDER_INDEX_VIEW_DEFAULT_TITLE="Smart Search Indexed Content"
 COM_FINDER_MAPS="Content Maps"
-COM_FINDER_MAPS_VIEW_DEFAULT_TITLE="Smart Search Content Mapping"
 COM_FINDER_MAPS_VIEW_DEFAULT_DESC="Display search content mapping."
+COM_FINDER_MAPS_VIEW_DEFAULT_TITLE="Smart Search Content Mapping"
 COM_FINDER_MENU_SEARCH_VIEW_DEFAULT_TEXT="The default search layout."
 COM_FINDER_MENU_SEARCH_VIEW_DEFAULT_TITLE="Search"
 COM_FINDER_SEARCH_VIEW_DEFAULT_TITLE="Smart Search results"
```
Comment (Quy) [inline administrator/language/en-GB/com_finder.sys.ini]: Swap lines 18-19.

Comment (tecpromotion) [reply administrator/language/en-GB/com_finder.sys.ini]: thanks @Quy

**Comment by jwaisner** [comment]
@tecpromotion Please provide test instructions for your PR.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27840">issues.joomla.org/tracker/joomla-cms/27840</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ed039c0570c9844a3ba57d5ae84c7f23a749eae9<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27840">issues.joomla.org/tracker/joomla-cms/27840</a>.</sub>

**Comment by tecpromotion** [comment]
> @tecpromotion Please provide test instructions for your PR.
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/27840](https://issues.joomla.org/tracker/joomla-cms/27840).

@jwaisner actually, no test instructions are necessary here, because the file has only been cleaned up. Nothing should change in the output. Please look at the summary of the pr, i just cleaned up the file.

**Review by alikon — APPROVED** [review]
sorry for my mistake
and thank you for cover my back :smile:

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on ed039c0570c9844a3ba57d5ae84c7f23a749eae9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27840">issues.joomla.org/tracker/joomla-cms/27840</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27840">issues.joomla.org/tracker/joomla-cms/27840</a>.</sub>

**Comment by Quy** [comment]
@tecpromotion Thanks!

**Pull request merged**